great_expectations/core/util.py

```diff
@@ -162,15 +162,27 @@ def determine_progress_bar_method_by_environment() -> Callable:
     return tqdm
 
 
-def convert_to_json_serializable(data):  # noqa: C901 - complexity 28
+def convert_to_json_serializable(data) -> dict:  # noqa: C901 - complexity 28
```

Comment:
This is an example of a method with added examples used in earlier API docs demos. Keeping it in this PR since these changes are illustrative.